AttitudeManager/Src/AM.cpp

```diff
@@ -2,13 +2,36 @@
 // Created by Anthony Luo on 2022-10-12.
 //
 #include "AM.hpp"
+#include "semphr.h"
 
 #include <array>
 #include <cstdlib>
 
 namespace AM {
+AttitudeManager::control_inputs_mutex = xSemaphoreCreateMutex();
```

Comment:
Should this be done in the AttitudeManager constructor? This seems like an odd place to define the mutex.

Reply:
I thought static variables aren't associated with a class instance? I don't think it would work to put these in the AM class constructor.